silverback/settings.py

```diff
@@ -31,6 +35,10 @@ class Settings(BaseSettings, ManagerAccessMixin):
     NEW_BLOCK_TIMEOUT: Optional[int] = None
     START_BLOCK: Optional[int] = None
 
+    # Used for persistent storage
+    PERSISTENCE_CLASS: Optional[str] = None
+    PERSISTENCE_URI: Optional[str] = None
```

Comment:
suggestion: from @fubuloubu: drop this.  Or rather, rename it to `SQLITE_PATH` and allow user-defined persistence classes to handle their own env vars/settings.